feature-libs/cart/quick-order/components/cart-quick-order-form/cart-quick-order-form.component.html

```diff
@@ -8,57 +8,119 @@
         <span class="cx-cart-quick-order-form-productID">
           <label class="cx-cart-quick-order-form-label">
             {{ 'quickOrderCartForm.productCodeLabel' | cxTranslate }}
-            <!-- TODO: (CXSPA-5953) Remove feature flags next major -->
```

Comment:
Same comments from `feature-libs/cart/base/components/cart-coupon/cart-coupon.component.html` apply to this template too.